person_capture/gui_app.py

```
@@ -10722,6 +10722,208 @@ def _save_hdr_archive_crop(
+        if not clean_hdr_path or not os.path.exists(clean_hdr_path):
+            return False, "windows_wic_unavailable:clean_hdr_source_missing", None, None
+
+        ref_tmp = ref_out_path + ".tmp.raw"
+        clean_tmp = clean_out_path + ".tmp.raw"
+        for out_path in (ref_out_path, clean_out_path):
+            out_dir = os.path.dirname(out_path)
+            if not out_dir:
+                continue
+            try:
+                ensure_dir(out_dir)
+            except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)